### DISCUSSION TITLE
Background color

### DISCUSSION THREAD

**Discussion by kamalkech** [neovide/neovide#1551] [Q&A]
i found the default background of neovide is black, so how can i change this color, i am using linux 
thank u

**Comment by MultisampledNight** [comment]
The behavior about highlights (which dictate "colors" in general in NeoVim) matches NeoVim exactly (and even goes beyond this, Neovide is just a dummy UI on top of NeoVim), so you can do the same thing as you'd do in NeoVim: Use the `highlight` command, namely modify the `Normal` highlight to set `guibg` to whatever color you want it to.

```vim
highlight Normal guibg=#000f80
```

**Comment by rameshsanth** [comment]
Use colorschemes to set up your colors. Literally hundreds of colorschemes are available for both dark and light backgrounds.

:help colorscheme

Here are some of the good ones.. everforest and cappuccin are my personal favorite.

Plug 'sainnhe/everforest'           " favorite. actively updated
Plug 'tomasr/molokai'
Plug 'sickill/vim-monokai'
Plug 'yuttie/hydrangea-vim'
" Plug 'flazz/vim-colorschemes'     " 100s of color schemes, beware slows down launch time
Plug 'lifepillar/vim-gruvbox8'
Plug 'iandwelker/rose-pine-vim'
if has("nvim")
  Plug 'catppuccin/nvim', {'as': 'catppuccin'}
endif
